python/guide/packages/gluon/loss/kl_divergence.md

```diff
@@ -0,0 +1,116 @@
+# Kullback-Leibler (KL) Divergence
+
+Kullback-Leibler (KL) Divergence is a measure of how one probability distribution is different from a second, reference probability distribution. Smaller KL Divergence values indicate more similar distributions and, since this loss function is differentiable, we can use gradient descent to minimize the KL divergence between network outputs and some target distribution. As an example, this can be used in Variational Autoencoders (VAEs), and reinforcement learning policy networks such as [Trust Region Policy Optimization (TRPO)](https://arxiv.org/abs/1502.05477).
+
+In MXNet Gluon, we can use [`KLDivLoss`](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.KLDivLoss.html) to compare categorical distributions. One important thing to note is that the KL Divergence is an asymmetric measure: order matters and we should compare our predicted distribution with our target distribution in that order. Another thing to note is that there are two ways to use [`KLDivLoss`](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.KLDivLoss.html) that depend on how we set `from_logits` (which has a default value of true).
```

Comment:
I would add to the statement about asymmetry: KL(P|Q) is not the same like KL(Q|P) though both measure the similarity between P and Q.

Reply:
Got a comment on it. Added `KL(P,Q) != KL(Q,P)`.